### PR TITLE
Lit utilities lists

### PR DESCRIPTION
**Pull request by haarsh24**
1. Added typography and lists

### CODE REVIEW

File: components/components.html [hunk 1]
```diff
@@ -48,10 +48,29 @@ <h2>
...
+
+
+
+
+
+
+
+
+
+
```
Comment (tushargoyal812) [inline components/components.html]: remove unnecessary space

File: components/lists.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.lists-item-stacked{
```
Comment (tushargoyal812) [inline components/lists.css:1]: add space before { this important for indentation.

File: components/utlis.css [hunk 3]
```diff
@@ -15,6 +15,29 @@
   --alert-danger:#f0814e;
 }
 
+
+
+
```
Comment (tushargoyal812) [inline components/utlis.css]: also here remove unnecessary space from here

**Review by tushargoyal812 — CHANGES_REQUESTED** [review]
everything is good you just need to take care of indentation.

File: components/lists.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    border:0.2rem solid var(--primary-color);
+    width:40rem;
+    border-radius: 0.5rem;
+    margin:0.1rem;
+    list-style: none;
+}
+
+.lists-item-stacked:hover{
+    background-color: var(--white);
+    color:var(--black);
```
Comment (xii-vi) [inline components/lists.css:11]: use --white-color instead of only white

**Review by xii-vi — APPROVED** [review]
Just a tiny suggestion, rest is fine.
You can merge.

**Pull request merged**